config/settings.env

```diff
@@ -61,3 +61,6 @@ export DISABLE_AUTH=true
 # 'error'.
 export LOG_LEVEL=info
 
+# Enable the Web AIM API server on port 9000 (default: disabled)
+# export ENABLE_WEBAPI=1
```

Comment:
Please re-run `make config`. This entry will get removed since it's not part of the [config struct](https://github.com/mk6i/retro-aim-server/blob/main/config/config.go#L43).